package.json

```diff
@@ -20,15 +20,15 @@
     "@types/lodash": "^4.14.186",
     "@types/node": "^14.14.33",
     "@types/showdown": "^2.0.0",
-    "colors": "1.4.0",
+    "colors": "^1.4.0",
```

Comment:
Please don't unpin the version. We had trouble with this.

Reply:
Reverted